src/main/kotlin/no/nav/familie/ef/mottak/task/ArkiverEttersendingTask.kt

```diff
@@ -2,29 +2,54 @@ package no.nav.familie.ef.mottak.task
 
 import io.micrometer.core.instrument.Counter
 import io.micrometer.core.instrument.Metrics
-import no.nav.familie.ef.mottak.repository.EttersendingRepository
+import no.nav.familie.ef.mottak.repository.domain.Ettersending
 import no.nav.familie.ef.mottak.service.ArkiveringService
+import no.nav.familie.ef.mottak.service.EttersendingService
+import no.nav.familie.kontrakter.felles.BrukerIdType
+import no.nav.familie.kontrakter.felles.journalpost.Bruker
 import no.nav.familie.prosessering.AsyncTaskStep
 import no.nav.familie.prosessering.TaskStepBeskrivelse
 import no.nav.familie.prosessering.domene.Task
 import no.nav.familie.prosessering.domene.TaskRepository
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
+import org.springframework.web.client.HttpClientErrorException
 
 @Service
 @TaskStepBeskrivelse(taskStepType = ArkiverEttersendingTask.TYPE, beskrivelse = "Arkiver ettersending")
 class ArkiverEttersendingTask(
     private val arkiveringService: ArkiveringService,
     private val taskRepository: TaskRepository,
-    private val ettersendingRepository: EttersendingRepository
+    private val ettersendingService: EttersendingService
 ) : AsyncTaskStep {
 
-    val logger: Logger = LoggerFactory.getLogger(this::class.java)
+    val logger: Logger = LoggerFactory.getLogger(javaClass)
+    val secureLogger: Logger = LoggerFactory.getLogger("secureLogger")
     val antallEttersendinger: Counter = Metrics.counter("alene.med.barn.journalposter.ettersending")
 
     override fun doTask(task: Task) {
-        val journalpostId = arkiveringService.journalførEttersending(task.payload)
+        val journalpostId: String = try {
+            arkiveringService.journalførEttersending(task.payload)
```

Comment:
Kan ikke `journalførEttersending` håndtere denne feilhåndteringen? Nå har både arkiveringService og denne tasken avhengighet til EttersendingService for å hente ut ettersendingen